File: src/codes/Users_Manual/gain/group_0.cs
~using AUTD3Sharp.Utils;
~using AUTD3Sharp.Gain;
~var x = 0.0f;
~var y = 0.0f;
~var z = 0.0f;
var g = new Group(dev => tr => tr.Idx <= 100 ? "null" : "focus")
          .Set("null", new Null())
          .Set("focus", new Focus(new Vector3(x, y, z)));

File: src/codes/Users_Manual/gain/group_0.rs
# use autd3::prelude::*;
# #[allow(unused_variables)]
# fn main() {
# let x = 0.;
# let y = 0.;
# let z = 0.;
let g = Group::new(|_dev| |tr| match tr.idx() {
                0..=100 => Some("null"),
                101..=200 => Some("focus"),
                _ => None,
            })
            .set("null", Null::new())
            .set("focus", Focus::new(Vector3::new(x, y, z)));
# }